src/Control/Carrier/FossaApiClient/Internal/FossaAPIV1.hs

```diff
@@ -947,7 +947,7 @@ archiveBuildUpload' apiOpts rebuild archive = context ("archive: '" <> toText ar
   (baseUrl, baseOpts) <- useApiOpts apiOpts
 
   let opts = "dependency" =: True <> "rawLicenseScan" =: True
-  let archiveProjects = ArchiveComponents [archive] rebuild FileUploadFullContent
```

Comment:
Based on the previous code: https://github.com/fossas/fossa-cli/pull/1423/files#diff-a812ed3f7085f40e35cb15511136c5dea108406ba8c21818dde5a4029b9302d5L942

I've changed this to FileUpload  to `FileUploadMatchData`

Reply:
Since this is an archive upload, why isn't `FullContent` appropriate here?

Reply:
Ah, I changed this to be `FileUploadMatchData` due to the fact that `archiveProjects` was set to `FullFileUploads False` previously. But after looking at Scott's comments on your initial pr, it seems that this was intended bc we shouldn't ever do an archive upload without full files. 

That's my fault, changing this back to `FileUploadFullContent`